### PR TITLE
Show your own NFT in “for sale” view

### LINKED ISSUE #521: Show your own NFT in “for sale” view

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes #521 

Added owner_badge on collectible card.

### CODE REVIEW

File: src/components/erc721/collectibles/collectibles_all.tsx [hunk 1]
```diff
@@ -159,7 +161,7 @@ export class CollectiblesAll extends React.Component<Props> {
 
 const allMapStateToProps = (state: StoreState): StateProps => {
     return {
-        collectibles: getOtherUsersCollectibles(state),
+        collectibles: getAllCollectibles(state),
```
Comment (unjapones) [inline src/components/erc721/collectibles/collectibles_all.tsx]: Posted [a question](https://github.com/0xProject/0x-launch-kit-frontend/issues/521#issuecomment-499187489) in the corresponding issue about the criteria that the collectibles should meet here.

File: src/components/erc721/collectibles/collectible_card.tsx [hunk 2]
```diff
@@ -56,19 +51,66 @@ const Title = styled.h2`
...
-            to={`${ERC721_APP_BASE_PATH}/collectible/${props.id}`}
-        >
-            <ImageWrapper color={color} image={image}>
-                <PriceBadge price={price} />
-            </ImageWrapper>
-            <Title>{name}</Title>
-        </CollectibleCardWrapper>
-    );
+interface OwnProps {
+    color: string;
```
Comment (unjapones) [inline src/components/erc721/collectibles/collectible_card.tsx]: We can probably simplify the interface of this component and just pass the `Collectible`, which has the id, image, name and owner; I believe.

File: src/components/erc721/collectibles/collectible_card.tsx [hunk 3]
```diff
@@ -56,19 +51,66 @@ const Title = styled.h2`
...
+    id: string;
+    image: string;
+    name: string;
+    price: BigNumber | null;
+    onClick?: (e: any) => void;
+    currentOwner: string;
+    mustShowCollectibleOwnerBadge?: boolean;
```
Comment (unjapones) [inline src/components/erc721/collectibles/collectible_card.tsx]: Req: according to [this comment](https://github.com/0xProject/0x-launch-kit-frontend/issues/521#issuecomment-499190843) we will always have to show the badge, so the flag is not needed anymore.

File: src/components/erc721/collectibles/collectibles_all.tsx (hunk 1 above)
Comment (unjapones) [reply src/components/erc721/collectibles/collectibles_all.tsx]: It looks like we should revert to the `getOherUsersCollectibles` and append the current user's collectibles that **are on sale**. Check [this comment](https://github.com/0xProject/0x-launch-kit-frontend/issues/521#issuecomment-499198412) and the answer below.

File: src/components/erc721/collectibles/collectible_card.tsx (hunk 2 above)
Comment (mariano-aguero) [reply src/components/erc721/collectibles/collectible_card.tsx]: Done in this [commit](https://github.com/0xProject/0x-launch-kit-frontend/pull/531/commits/da3912ff9509402867b4632ff52c2181a4015ab4)

File: src/components/erc721/collectibles/collectible_card.tsx (hunk 3 above)
Comment (mariano-aguero) [reply src/components/erc721/collectibles/collectible_card.tsx]: Done in this [commit](https://github.com/0xProject/0x-launch-kit-frontend/pull/531/commits/5397939617dbd5a4cc53c523f75526f67c793a54)

File: src/components/erc721/collectibles/collectibles_all.tsx (hunk 1 above)
Comment (mariano-aguero) [reply src/components/erc721/collectibles/collectibles_all.tsx]: Done in this [commit](https://github.com/0xProject/0x-launch-kit-frontend/pull/531/commits/827eb390cc33bff37b3e81ce27192aa819914c4e)

**Pull request merged**